core/common/src/main/java/com/together/study/common/type/study/StudyTier.kt

```
@@ -0,0 +1,25 @@
+    SLIVER_3(title = "Sliver3"),
+    GOLD_1(title = "Gold1"),
+    GOLD_2(title = "Gold2"),
+    GOLD_3(title = "Gold3"),
+    MASTER(title = "Master"),
+    LEGEND(title = "Legend");
+
+    companion object {
+        fun get(studyTier: String): StudyTier {
+            return try {
+                StudyTier.valueOf(studyTier.uppercase())
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)